Lib/test/test_file.py

```diff
@@ -242,7 +242,7 @@ def testIteration(self):
         for methodname, args in methods:
             f = self.open(TESTFN, 'rb')
             if next(f) != filler:
-                self.fail, "Broken testfile"
+                self.fail("Broken testfile")
```

Comment:
Wouldn't be better to use `assertEqual()`?

Reply:
Sure, looks a bit neater.